.rubocop.yml

```diff
@@ -22,14 +20,19 @@ AllCops:
     - 'config/puma.rb'
     - 'db/migrate/*'
     - '**/Capfile'
+    - 'tmp/**/*'
+    - 'vendor/**/*'
+    - 'public/**/*'
+    - 'node_modules/**/*'
   TargetRubyVersion: 2.5
   TargetRailsVersion: 6.1
   UseCache: true
+  DisabledByDefault: true
 
 Rails:
-  Enabled: true
+  Enabled: false
```

Comment:
Curious if I'm following correctly. If we disable rules by default, do we need to have any of these where we explicitly set `Enabled: false` ?

Reply:
We do not, will remove